src/components/TeaserEmbedComment/index.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+import * as Comment from '../Discussion/Internal/Comment'
+import { parse } from '@orbiting/remark-preset'
```

Comment:
todo: refactor this out of the client – the remark parser is quite heavy for the client

Reply:
184kb -> 239kb
<img width="1171" alt="Screenshot 2021-05-12 at 19 55 05" src="https://user-images.githubusercontent.com/410211/118021956-18d66900-b35c-11eb-8510-d4a54f8de108.png">

Reply:
wow. ok, yeah. @patrickvenetz and I had talked about maybe delegating the parsing to the backend (which it already does for other comments). so that's a potential solution – unless you had something schlauer in mind.

Reply:
one possible solution: do it in publikator frontend – which already ships with the parser afaik and is less code size sensitive – and save the parsed ast to the node data.